src/components/logout/logout-controller.ts

```diff
@@ -3,5 +3,6 @@ import { Request, Response } from "express";
 export async function logoutGet(req: Request, res: Response): Promise<void> {
   const idToken = req.session.user.tokens.idToken;
   req.session.destroy();
+  res.cookie("lo", "true");
```

Comment:
is the value for this cookie reset on login?

Reply:
Directly, no, but in practice yes. It's reset the first time the auth middleware is called with a valid, authenticated user session - which would be after a successful login and the first time the user tries to access an auth-protected page like the main account management page. Does that work for you?: https://github.com/alphagov/di-authentication-account-management/blob/f3136dbbea03b9618e24a9344496c8bed66f05d0/src/middleware/requires-auth-middleware.ts#L17

Reply:
Ah ok. Yep. That makes sense.